.github/workflows/e2e.yml

```diff
@@ -0,0 +1,46 @@
+name: "ZITADEL e2e Tests"
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+    inputs:
+      releaseversion:
+        description: 'Release version to test'
+        required: true
+        default: 'latest'
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set TAG env manual trigger
+        if: github.event_name == 'workflow_dispatch'
+        run: echo "RELEASE_VERSION=${{ github.event.inputs.releaseversion }}" >> $GITHUB_ENV
+      - name: Set TAG env on release
+        if: github.event_name == 'release'
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver: docker
+          install: true
+      - name: Set up Docker Image Layer Caching
+        uses: satackey/action-docker-layer-caching@v0.0.11
```

Comment:
Lets remove this since we have no experience with it.
We can create a new PR down the road for this

Reply:
done by @eliobischof

Reply:
removed